engine/src/graphic.cpp

```diff
@@ -745,15 +745,15 @@ void MCGraphic::draw_arrow(MCDC *dc, MCPoint &p1, MCPoint &p2)
 	dc->fillpolygon(pts, 3);
 }
 
-void MCGraphic::draw_lines(MCDC *dc, MCPoint *pts, uint2 npts)
+void MCGraphic::draw_lines(MCDC *dc, MCPoint *pts, uindex_t npts)
```

Comment:
Given that we are limiting the number of pts to 65535 - how come we had to change the prototype of draw_lines?